interview-questions.md

```diff
@@ -7,60 +7,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: I understand that Github is a website that holds code repositories and git holds the time line.
 
-  Researched answer:
+  Researched answer: GIT is a way to track changes in code and Github is the a is a hosting website that we use to hold and request code.
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: Boolean value is a true or false data type and it happens whenever you make a comparison between two values
 
-  Researched answer:
+  Researched answer: Is a primitive data type that evaluate true or false mainly with equality operators
```

Comment:
Yup, you got it. Just to expand, operators like relational/logical/equality return Boolean values.